README.md

```diff
@@ -15,51 +15,57 @@
 <h3 align="center">
   <a href="https://docs.aklivity.io/zilla/"><b>Documentation</b></a> &bull;
   <a href="https://docs.aklivity.io/zilla/latest/tutorials/quickstart/kafka-proxies.html"><b>Quickstart</b></a> &bull;
-  <a href="https://github.com/aklivity/zilla-examples"><b>Examples</b></a> &bull; 
+  <a href="https://github.com/aklivity/zilla-examples"><b>Examples</b></a> &bull;
   <a href="https://www.aklivity.io/blog"><b>Blog</b></a>  
 </h3>
 
+# A multi-protocol edge and service proxy designed for event-driven architectures
 
-##  A multi-protocol edge and service proxy, designed for event-driven architectures
-Zilla abstracts Apache Kafka® for web applications, IoT clients and non-Kafka microservices. With Zilla, Kafka topics can be securely and reliably exposed via a user-defined `REST`, `Server-Sent Events (SSE)`, `MQTT`, or `gRPC` API.
+Zilla abstracts Apache Kafka® for web applications, IoT clients, and non-Kafka microservices. With Zilla, Kafka topics can be securely and reliably exposed via a user-defined `REST`, `Server-Sent Events (SSE)`, `MQTT`, or `gRPC` API.
```

Comment:
```
Zilla abstracts Apache Kafka® for web applications, IoT clients, and microservices.
```

I'm not sure what a `non-Kafka microservice` is. My guess is microservices that don't connect to Kafka through an SDK or API. Maybe we can just say `microservice` or try to clarify it some more?